plugin/integration_tests/test_plugin.py

```diff
@@ -52,22 +51,19 @@ def test_create_instance(self):
                                   config['scope'],
                                   ctx.logger)
         instances = gcp.list_instances()
-        item = utils.get_item_from_gcp_response('name', 'testnode', instances)
-        self.assertIsNone(item)
 
+        base = len(instances['items'])
         ctx.logger.info('Install workflow')
         # execute install workflow
         self.env.execute('install', task_retries=0)
 
         ctx.logger.info('Check instance number')
         instances = gcp.list_instances()
-        item = utils.get_item_from_gcp_response('name', 'testnode', instances)
-        self.assertIsNotNone(item)
+        self.assertEqual(len(instances['items']), base + 1)
```

Comment:
can we also assert the details of the instance we found is what we expect it to be?